src/main/java/org/irods/Util/Property.java

```diff
@@ -0,0 +1,19 @@
+package org.irods.Util;
+
+/**
+ * enum for modify() in zones endpoint
+ */
+public enum Property {
```

Comment:
`Property` is a little too generic.

Let's go with `ZoneProperty` so it's clear what these enums are for.

Reply:
Makes sense, done.